### ISSUE TITLE
Revision request page reported as not accessible

### ISSUE THREAD

**Issue by joemull** [BirkbeckCTP/janeway#3008]
**Describe the bug**
A partially sighted user reported not being able to find the way to accept a peer review request.

**Janeway version**
1.4.2

**To Reproduce**
1. Go to the [Wave accessibility checker](https://wave.webaim.org/)
2. Put in a one-click-access review request link
3. See errors: 12 low contrast errors, 4 missing form labels, and 4 empty links.

**Expected behavior**
The page should not have the errors that are currently showing. We should be able to debug the page by altering the template attributes involved.

## Front-end Issues

**Desktop (please complete the following information):**
 - Browser: Firefox
 - Version 102

<img width="364" alt="Screen Shot 2022-07-15 at 11 25 08" src="https://user-images.githubusercontent.com/47217308/179205345-7aa77ae6-d854-456a-8065-9a0c9aef8608.png">

**Comment by StephDriver** [comment]
tldr:  Dev Ready, Medium T-shirt, and assign to me.

I think this is Dev-ready, just a question of setting a priority and adding it to a project.  I would like to take it on, and feed what I find into the general A11y project planning - as errors here may help spot errors elsewhere.   While the errors above seem straightforward, my experience with automated testing makes me think that they may not be the same as whatever is causing the issue the user reported and there could be other factors too, so I want to put this in as a medium to give time for manual review and testing of the template (get it done once!).
